src/data/actions/assets.js

```diff
@@ -42,9 +46,12 @@ export const deleteAsset = (assetsParameters, assetId) =>
     clientApi.requestDeleteAsset(assetsParameters.courseId, assetId)
       .then((response) => {
         if (response.ok) {
-          dispatch(deleteAssetSuccess(assetId));
+          dispatch(deleteAssetSuccess(assetId, response, AssetsConstants.SUCCESS_FILE_DELETE));
```

Comment:
Should pose the question -- do we want to do this in this manner where the `SUCCESS_FILE_DELETE` action type is used in two different reducers, or do we want two dispatch separate `dispatch` calls and create a new `assetXHRSuccess` / `ASSET_XHR_SUCCESS` type? I feel like the way I did it isn't recommended even if it does technically work.  It might also be more clear if we do two separate dispatch calls? @arizzitano specific react/redux framework question